crypten/mpc/mpc.py

```diff
@@ -1231,16 +1325,12 @@ def set(self, enc_tensor):
 }
```

Comment:
This is the location of `__xor__`, `__or__`, and `__and__`. These are how one should call binary xor and and functions above, and we should remove them from here.